libs/designsystem/src/lib/components/form-field/_form-field-inputs.shared.scss

```diff
@@ -54,3 +54,10 @@ $form-field-label-height: 24px;
     color: get-text-color('semi-dark');
   }
 }
+
+:host-context(kirby-item) {
+  border-radius: 0 !important;
+  box-shadow: none !important;
+  padding: 1px 2px !important;
+  width: auto !important;
+}
```

Comment:
!important used to force overrule &:not(.borderless) - Not a pretty solution, need some input on this :)

Reply:
Yes, it's not very pretty, but I don't have a good suggestion for fixing it otherwise... 🤔 
Maybe add a `TODO` comment about it, and we'll fix it later?